plugins/module_utils/mso.py

```diff
@@ -788,7 +788,7 @@ def query_objs(self, path, key=None, api_version="v1", **kwargs):
         found = []
         objs = self.request(path, api_version=api_version, method="GET")
 
-        if objs == {} or objs == []:
+        if objs == {} or objs == [] or not objs:
```

Comment:
is [] or {} not the same as not?
```suggestion
        if not objs:
```